src/bin.ts

```diff
@@ -7,9 +7,26 @@ import runVrt from './index';
 const cwd = (argv.cwd as string) || process.cwd();
 const output = (argv.output as string) || path.resolve(cwd, 'result');
 const teamcity = !!argv.teamcity;
+const threshold = argv.threshold as number;
+const includeAA = argv.threshold as boolean;
+const alpha = argv.threshold as number;
+const aaColor = argv.threshold as [number, number, number];
+const diffColor = argv.threshold as [number, number, number];
+const diffColorAlt = argv.threshold as [number, number, number];
```

Comment:
I am not sure how yargs can understand an array of numbers in arguments? Have you tested this?